db/system_keyspace.cc

```diff
@@ -2149,9 +2255,6 @@ std::vector<schema_ptr> system_keyspace::all_tables(const db::config& cfg) {
                     topology(), cdc_generations_v3(), topology_requests(), service_levels_v2(),
     });
 
-    auto auth_tables = db::system_auth_keyspace::all_tables();
```

Comment:
The order doesn't matter?

Reply:
it doesn't , tables don't depend on each other